src/commands/ext-configure.ts

```diff
@@ -48,21 +48,27 @@ export default new Command("ext:configure <extensionInstanceId>")
       const paramSpecWithNewDefaults = paramHelper.getParamsWithCurrentValuesAsDefaults(
         existingInstance
       );
-      const removedLocations = _.remove(paramSpecWithNewDefaults, (param) => {
-        return param.param === "LOCATION";
+      const immutableParams = _.remove(paramSpecWithNewDefaults, (param) => {
+        return param.immutable || param.param === "LOCATION";
+        // TODO: Stop special casing "LOCATION" once all official extensions make it immutable
       });
-      const currentLocation = _.get(existingInstance, "config.params.LOCATION");
+
       const params = await paramHelper.getParams(
         projectId,
         paramSpecWithNewDefaults,
         options.params
       );
-      if (removedLocations.length) {
+      let immutableParamsText = "";
+      for (const immutableParam of immutableParams) {
+        const currentValue = _.get(existingInstance, "config.params." + immutableParam.param);
+        immutableParamsText += `param: ${immutableParam.param}, value: ${currentValue}\n`;
+        params[immutableParam.param] = currentValue;
+      };
+      if (immutableParamsText) {
         logger.info(
-          `Location is currently set to ${currentLocation}. This cannot be modified. ` +
-            `Please uninstall and reinstall this extension to change location.`
+          `The following params are immutable:\n${immutableParamsText}These cannot be modified. ` +
```

Comment:
Looks like there's a missing period and space between the param placeholder and the "These".

Also, quick nit question- does this code/language take into account if only ONE param is in the lists of immutables?  (plural phrasing vs singular phrasing)

Reply:
To the missing period/space: ${immutableParamsText} always ends in '/n' if this is printed, so theres no need - check out the picture in the description to see what it ends up looking like

To the question about plurals: It doesn't right now, but its pretty easy to make it correct, so ill switch it over

Reply:
Ah!  mis-read the combo of sentences - sorry!  Yes, this is fine to just break the line and not have a period.

Quick fix request: the "To set a different value for this params..." should be "To set a different value for this param..."